editoast/src/views/rolling_stocks/mod.rs

```diff
@@ -598,26 +654,24 @@ pub mod tests {
     use crate::modelsv2::prelude::*;
     use crate::modelsv2::rolling_stock_model::RollingStockModel;
     use crate::views::rolling_stocks::rolling_stock_form::RollingStockForm;
+    use crate::views::test_app::TestApp;
     use crate::views::test_app::TestAppBuilder;
-    use crate::views::tests::create_test_service;
 
-    fn rolling_stock_create_request(rolling_stock_form: &RollingStockForm) -> Request {
-        TestRequest::post()
-            .uri("/rolling_stock")
-            .set_json(rolling_stock_form)
-            .to_request()
-    }
+    impl TestApp {
```

Comment:
Is there a reason to add this impl TestApp?
Why not just use this:
```rust
let request = app.post("/rolling_stock").json(rolling_stock_form);
let request = app.get(format!("/rolling_stock/{rolling_stock_id}").as_str());
```

Reply:
We're trying to keep the tests structure as close as possible to the actix ones in order to lower the risk of introducing bugs.